podvm/addons/nvidia_gpu/setup.sh

```diff
@@ -39,7 +41,7 @@ if  [[ "$PODVM_DISTRO" == "rhel" ]]; then
     dnf install -q -y kernel-devel-"$(uname -r)" kernel-headers-"$(uname -r)"
 
     dnf install -q -y nvidia-container-toolkit
-    dnf -q -y module install nvidia-driver:latest
+    dnf -q -y module install nvidia-driver:${NVIDIA_DRIVER_VERSION}/default
```

Comment:
is this valid versioning?

Reply:
It's version and stream. `default` is the stream